lambdas/tests/unit/handlers/test_create_document_reference_handler.py

```diff
@@ -30,18 +35,82 @@
     },
 }
 
+def setEnvironmentVariables():
+    os.environ["DOCUMENT_STORE_DYNAMODB_NAME"] = MOCK_DYNAMODB_ARF
+    os.environ["DOCUMENT_STORE_BUCKET_NAME"] = MOCK_BUCKET_ARF
+    os.environ["LLOYD_GEORGE_DYNAMODB_NAME"] = MOCK_DYNAMODB_LG
+    os.environ["LLOYD_GEORGE_BUCKET_NAME"] = MOCK_BUCKET_LG
 
+    
 @pytest.fixture
-def event():
+def arf_type_event():
     api_gateway_proxy_event = {
+        "queryStringParameters": {"documentType": "arf"},
         "body": '{"subject": {"identifier": {"value": "test"}}, '
         '"content": [{"attachment": {"contentType": "test"}}], '
         '"description": "test"}'
     }
     return api_gateway_proxy_event
 
+@pytest.fixture
```

Comment:
I think abbas wanted fixtures that are used in multiple places to be moved to conftest.py, would this be a candidate for that?